content/committees/specification/tckprocess/index.md

```diff
@@ -412,3 +423,11 @@ is not a mechanism to attempt to handle implementation specific issues.
 The only time tests may be added to a TCK are in a major or minor
 release. A service release which updates the exclude list MUST not have
 test additions or changes.
+
+## Process for releasing a point revision of a TCK
+
+The process for releasing a point revision of a TCK entails filing an issue in the [jakartaee/specifications repository with the following details](https://github.com/jakartaee/specifications):
+
+-   Link to the TCK release to be published.
+-   Updated TCK links in the specification's _index.md file.
```

Comment:
Let's pretend that this section also applies to service releases (x.y.z).  When is it required for an implementation to move up to the latest x.y.z version of the TCK?  Maybe that's spelled out elsewhere in the TCK process.  I'm only reviewing the updates right now.  But, I thought we had some grace period defined to allow an organization to continue using the previous versions of the TCK up to a certain point in time.  This allows an existing test effort to continue to completion without having to start over with a new TCK revision.

Reply:
> Let's pretend that this section also applies to service releases (x.y.z). When is it required for an implementation to move up to the latest x.y.z version of the TCK? 

I have been thinking about how this is undefined for Jakarta EE and we should answer this question.  The cost of continually being able to process TCK challenges for every released EE version forever, is too great.  

> Maybe that's spelled out elsewhere in the TCK process. 

It is not spelled out anywhere that I am aware of.

> I'm only reviewing the updates right now. But, I thought we had some grace period defined to allow an organization to continue using the previous versions of the TCK up to a certain point in time. 

I haven't heard about that for Jakarta EE.  

> This allows an existing test effort to continue to completion without having to start over with a new TCK revision.

Are you saying that they could continue to use the Jakarta EE 9.0.0 Platform TCK forever without having to start over with the latest 9.0.x Platform TCK version?

Reply:
We should decide soon how far back we support TCK challenges for, so that it is clear when we require moving up to the latest TCK revision.